### ISSUE TITLE
Validation result doesn't convergent for "deep fusion"

### ISSUE THREAD

**Issue by chenyez** [kujason/avod#41]
Hi, thank you for sharing your code, we are doing relative research and it's very helpful for us!

Now we train the network using pyrimid_cars_with_aug_example.config, with fusion type "deep". After the validation, the best car_detection_3D results appears at around the iteration 52500 checkpoint, which is:
car_detection_3D : [52500, 120000, 70000, 100000, 102500]

In 52500 checkpoint, the result is:  
car_detection_3D AP: 84.555695 74.843224 68.156281

Moreover, the result vibrates at the later checkpoints, even decrease to (car_detection_3D AP: 77.260490 68.040474 67.316147) at 117500 checkpoint,  as you can see in the attachment figure.

Since it is our first time to train such a large network, my question is: is it a normal thing to have the best performance at 52500 checkpoint? Can we say that the result has already converged? Thanks and looking forward to your reply!
![capture](https://user-images.githubusercontent.com/30028865/38968848-d6ab5b16-4352-11e8-8e7b-2106731df2d3.PNG)

**Comment by griffintin** [comment]
@chenyez 

I tried to train using pyrimid_cars_with_aug_example.config, however, with fusion type "early". No matter with view drop on or off, both results do not converge, fluctuations among iterations happens and in the range of 77% - 84% for car 3D detection.
Have you tried early fusion? and what is the result like?

@melfm @kujason 
how can we get a more stable training?  
Any advices/comments will be greatly appreciated.

**Comment by griffintin** [comment]
@asharakeh 
Thank you for your reply.
I also checked the loss itself using tensorboard, following are the output figures. 
And best AP (84%, 74%, 68%) is achieved at iter=125000, not after 125000.
I expect loss should be more "stable"... But I don not know how to do.
And, VGG_FPN base network is trained from random values, not initialized by, like ImageNet weights.


![20180426_avod_fpn_loss](https://user-images.githubusercontent.com/31429693/39285548-eddbc24c-4953-11e8-87aa-d6864e40bc79.png)

By the way, I want to know whether overfitting is happend already. So I tried to inference test images.  
Problem is that,testing is using fixed ground plane [0, -1, 0, 1.65], while the above training is using provided ground plane. Sadly, for tested images, almost no car could be detected.

Any advices wil be appreciated.
